app/presenters/idv/by_mail/letter_enqueued_presenter.rb

```diff
@@ -0,0 +1,47 @@
+module Idv
+  module ByMail
+    class LetterEnqueuedPresenter
+      include ActionView::Helpers::TranslationHelper
+      include Rails.application.routes.url_helpers
+
+      attr_reader :url_options
+
+      def initialize(idv_session, url_options:)
+        @pii = idv_session.pii_from_doc ||
```

Comment:
2 things:

1. I'm wondering if we can add more args to this method to avoid having to make `idv_session.user_session` public and also remove some indirection here? E.g.

```ruby
def initialize(idv_session:, current_user:, user_session:, url_options:)
```

2. Can this be refactored into `pii` method?

```ruby
def pii
@pii ||= # big list of different ways we discover pii
end

```

Reply:
Oh and 3. maybe we should add a `pii_from_user` method to `Idv::Session`, since we have this same flow session / dig stuff cropping up

Reply:
Changed all of the above.